src/Umbraco.Web.UI.Client/src/less/installer.less

```diff
@@ -285,45 +286,3 @@ select {
   margin-bottom: 40px;
 }
 
-#consentSlider {
-  .noUi-target {
-    background: linear-gradient(to bottom, @grayLighter 0%, @grayLighter 100%);
-    box-shadow: none;
-    border-radius: 20px;
-    height: 8px;
-    border: 1px solid @inputBorder;
-
-    &:focus,
-    &:focus-within {
-      border-color: @inputBorderFocus;
-    }
-  }
-
-  .noUi-handle {
-    cursor: grab;
-    border-radius: 100px;
-    border: none;
-    box-shadow: none;
-    width: 20px !important;
-    height: 20px !important;
-    right: -10px !important; // half the handle width
-    top: -1px;
-    background-color: @blueExtraDark;
-  }
-
-  .noUi-handle::before {
-    display: none;
-  }
-
-  .noUi-handle::after {
-    display: none;
-  }
-
-  .noUi-value {
-    cursor: pointer;
-  }
-
-  .noUi-pips-horizontal {
-    height: 40px;
-  }
-}
```

Comment:
@iOvergaard @nul800sebastiaan it seems this part hasn't been removed from installer.less as it still exists here: https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Web.UI.Client/src/less/installer.less#L312-L355

I guess it somehow was re-introduced in https://github.com/umbraco/Umbraco-CMS/commit/26d83381f24fa5b11d1564e283660be31e07f82b

Reply:
Thanks, @bjarnef . We are reverting the "merge gone wrong" in PR #12923